docs/src/man/getting_started.md

```diff
@@ -227,30 +427,4 @@ df = DataFrame(x = 1, y = 2)
 CSV.write(output, df)
 ```
 
-The behavior of CSV functions can be adapted via keyword arguments. For more information, use the REPL [help-mode](http://docs.julialang.org/en/stable/manual/interacting-with-julia/#help-mode) or checkout the online [CSV.jl documentation](https://juliadata.github.io/CSV.jl/stable/).
-
-## Loading a Classic Data Set
```

Comment:
I've removed this as it feels out of place (it's not used in this page). Other pages include code to load it anyway.